README.md

```diff
@@ -14,6 +14,7 @@ This Repository hosts various plugins relavant to OpenStack and Kubernetes Integ
 * [Manila CSI Plugin](/docs/using-manila-csi-plugin.md/)
 * [Manila Provisioner](/docs/using-manila-provisioner.md/)
 * [Barbican KMS Plugin](/docs/using-barbican-kms-plugin.md/)
+* [Magnum Auto Healer](/docs/using-magnum-auto-healer.md/)
 
```

Comment:
@lingxiankong should we add flexvolume here and mark as deprecated explicitly?

Reply:
Done